skyvern/forge/sdk/routes/credentials.py

```diff
@@ -111,6 +121,74 @@ async def send_totp_code(
     )
 
 
+@legacy_base_router.get(
```

Comment:
we don't have to support legacy routes for this endpoint. let's remove two `legacy_base_router` decorators here